src/test/kotlin/apply/application/ApplicantServiceTest.kt

```diff
@@ -52,6 +52,49 @@ internal class ApplicantServiceTest {
         )
     }
 
+    @Nested
+    inner class Find {
+        @BeforeEach
+        internal fun setUp() {
+            val applicantId = 1L
+
+            slot<Long>().also { slot ->
+                every { applicationFormRepository.findByRecruitmentIdAndSubmittedTrue(capture(slot)) } answers {
+                    listOf(createApplicationForm(recruitmentId = slot.captured))
+                }
+            }
+
+            every { cheaterRepository.findAll() } returns listOf(Cheater(applicantId))
+            slot<Iterable<Long>>().also { slot ->
+                every { applicantRepository.findAllById(capture(slot)) } answers {
+                    slot.captured.map { createApplicant(id = it) }
+                }
+            }
+
+            slot<String>().also { slot ->
+                every { applicantRepository.findAllByKeyword(capture(slot)) } answers {
+                    listOf(createApplicant(name = slot.captured, id = applicantId))
+                }
+            }
```

Comment:
```suggestion
            every { applicantRepository.findAllByKeyword(any()) } answers {
                listOf(createApplicant(name = it.invocation.args[0] as String, id = applicantId))
            }
```

마침 answer를 사용하기 적당한 예제가 있네요. answer를 사용하면 좀 더 간단하게 표현할 수 있습니다 😄